changelog.md

```diff
@@ -2,6 +2,9 @@
     - Drop support for GHC < 8.4, require base > 4.11.
     - Bump `cabal-version` to 2.4.
     - Add `Ord` instance for `PlaylistName`, `Path`, and `Value`
+    - Add support for MPD 0.21 style filters to the `Query` type,
```

Comment:
Can you bump the package version to 0.10.0 (since we have some breaking changes at type level. Correct me if I'm wrong) and also update the `unreleased` heading in this file ? I will do a new release once the MR is merged.

Reply:
That's the nice thing, this isn't actually a breaking change. Because the data constructors for `Query` aren't exported, and the combinators default to MPD 0.20 style filters if they can, this both doesn't break existing code and still works for MPD 0.20 and under unless you use the new functions (since the only way to create `Query`'s is with the combinators). As a sanity check, I have compiled [vimus](https://github.com/vimus/vimus) against this branch and it works without errors.

Reply:
Nice! In that case, I guess we can bump it to 0.9.3.0. :-)

Reply:
Done!